ompi/request/grequest.c

```diff
@@ -216,11 +218,15 @@ int ompi_grequest_invoke_query(ompi_request_t *request,
         if (g->greq_funcs_are_c) {
             rc = g->greq_query.c_query(g->greq_state, status);
         } else {
+#if OMPI_BUILD_FORTRAN_BINDINGS
```

Comment:
I don't think that `g->greq_funcs_are_c` can ever be false if we didn't build Fortran support.

Is there a reason to #if this block out?  If not, it would be good to keep it always compiled, if for no other reason than to ensure someone doesn't accidentally introduce a compile error in the #if or #else block.